File: static/code/sample/tethered/gdl-securrent-a.cs
// In this sample:
// Display current (A)

using GHIElectronics.DUELink;

var availablePort = DUELinkController.GetConnectionPort();
var duelink = new DUELinkController(availablePort);

float ReadCurrent() {
    return duelink.Engine.ExecuteCommand("Current()");
}

while (true) {   
    Console.WriteLine($"Current: {ReadCurrent()} A");    

    Thread.Sleep(1000);
}

File: static/code/sample/tethered/gdl-securrent-a.py
# In this sample:
# Display current (A)

import time
from DUELink.DUELinkController import DUELinkController

availablePort = DUELinkController.GetConnectionPort()
duelink = DUELinkController(availablePort)

# Methods
def ReadCurrent():
    return float(duelink.Engine.ExecuteCommand("Current()"))

while True:
    print(f"Current: {ReadCurrent()} A")
    time.sleep(1)  # 1000 ms